.github/workflows/prepare-release.yaml

```diff
@@ -0,0 +1,73 @@
+name: Prepare release
+
+on:
+  workflow_dispatch:
+    inputs:
+      release-type:
+        type: choice
+        description: |
+          Release type
+          Use prepatch, preminor, premajor for the first "pre", to increment pre value use prerelease.
+        options:
+          - prerelease
+          - patch
+          - minor
+          - major
+          - prepatch
+          - preminor
+          - premajor
+        required: true
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    env:
+      RELEASE_TYPE: ${{ github.event.inputs.release-type }}
+    steps:
+      - uses: actions/checkout@v3
+      # Configure Git
+      - name: Git configuration
+        run: |
+          git config user.name Release-Workflow
+          git config user.email github-actions@github.com
```

Comment:
Not a big deal, but seems like there was some more detailed way of setting the email: https://github.com/orgs/community/discussions/26560